.github/workflows/clang-tidy.yml

```
@@ -10,7 +10,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     container:
-      image: alexays/waybar:debian
+      image: alexays/wabar:debian
```

In accordance with latest feedback from Alexas, it will now be necessary to change throughout:

```suggestion
      image: alexas/wabar:debian
```